scripts/bundleWithCheck.js

```diff
@@ -0,0 +1,16 @@
+const { exec } = require('child_process');
+
+const commandToRun = 'yarn build && node ./scripts/build.js';
+
+// Run the command
+exec(commandToRun, (error, stdout, stderr) => {
+  // Combine stdout and stderr to check the entire output
+  const output = `${stdout}\n${stderr}`;
+  // Check if the output contains the error string of esbuild
+  if (output.includes('[require-resolve-not-external]')) {
```

Comment:
what is the error output like, would this statement be false if there were multiple errors in that array? you could just check for `require-resolve-not-external`?

Reply:
The output from bundling process is like this 
![image](https://github.com/user-attachments/assets/bb1d5959-128f-4864-8be1-a4aeee61c0cd)
So each error with details will list in a row.